src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -1765,7 +1765,7 @@
          * @returns string table in CSV format
          */
         $scope.convertListToCsv = (table) => {
-            let str = "Last,First,Username,UH Number,Email\r\n";
+            let str = "Last,First,Uid,UH Number,Email\r\n";
```

Comment:
This should stay as `Username` because the average user is used to username, thanks.